### PR TITLE
feat: add new tower-hyper based openapi client

### PR DESCRIPTION
**Pull request by tiagolobocastro**
feat: use openapi-generator with tower support

New Code Generator: https://github.com/openebs/openapi-generator/commits/rust_mayastor

-------------
chore: autogenerate the openapi code on first build

-------------
chore: update openapi generated code

It now includes different features for actix client/server and tower client.

-------------
fix: update existing code to new openapi generated code

-------------
feat(kubectl-plugin): start using the tower-based client

-------------
feat(kubectl-plugin): enable jaegertracing

### CODE REVIEW

**Review by paulyoong — APPROVED** [review]

File: openapi/src/apis/actix_server.rs [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+        f.debug_struct("RestError")
+            .field("status_code", &self.status_code)
+            .field("error_response", &self.error_response)
+            .finish()
+    }
+}
+
+impl<T: Debug + Serialize> Display for RestError<T> {
+    fn fmt(&self, _: &mut Formatter<'_>) -> fmt::Result {
+        unimplemented!()
```
Comment (gila) [inline openapi/src/apis/actix_server.rs:43]: this needs to be fixed, otherwise any print of the error might crash.

Comment (tiagolobocastro) [reply openapi/src/apis/actix_server.rs]: Done

**Comment by tiagolobocastro** [comment]
bors try

**Comment by tiagolobocastro** [comment]
bors merge

**Pull request merged**